packages/gateway/src/features/user/usersAPI.ts

```diff
@@ -63,7 +64,8 @@ export class UsersAPI extends OpenCRVSRESTDataSource {
       return await response
     } catch (e) {
       // Don't need to throw errors if unauthorized error is found for no user with this mobile
-      if (e instanceof AuthenticationError) return null
+      if (e instanceof GraphQLError && e.extensions.code === 'UNAUTHENTICATED')
+        return null
```

Comment:
The fix makes sense in it's context, but generally doesn't it feel a bit weird that `getUser` endpoints get called if a user setting is being changed 🤔

Reply:
True, searchUsers is probably the right query here as that's what we are ultimately doing, searching for users with a given number/email

Reply:
Think ideally we'd try to call the "updateXyz" whatever and let it throw a conflict error from backend if there's a clashing key such as phonenumber or email. I think now the frontend unnecessarily handles the validation, even though we would need to handle it on the backend for security any way.

Reply:
Yeeea, why does the client check if someone with the email already exists and not the backend 🤔 This might a case where the UI design has affected the technical implementation a little.